contracts/test-utils/test/order_hash_test.ts

```diff
@@ -64,19 +63,4 @@ describe('Order hashing', () => {
             expect(() => orderHashUtils.getOrderHashHex(orderWithInvalidtakerFormat)).to.throw(expectedErrorMessage);
         });
     });
-    describe('#isValidOrderHash', () => {
-        it('returns false if the value is not a hex string', () => {
-            const isValid = orderHashUtils.isValidOrderHash('not a hex');
-            expect(isValid).to.be.false();
-        });
-        it('returns false if the length is wrong', () => {
-            const isValid = orderHashUtils.isValidOrderHash('0xdeadbeef');
-            expect(isValid).to.be.false();
-        });
-        it('returns true if order hash is correct', () => {
-            const orderHashLength = 65;
-            const isValid = orderHashUtils.isValidOrderHash(`0x${Array(orderHashLength).join('0')}`);
-            expect(isValid).to.be.true();
-        });
-    });
```

Comment:
Let's move the tests over to the `contracts-coordinator` as well.

Reply:
As discussed, these tests _were_ kept, but they got moved to `@0x/contracts-test-utils` rather than `contracts-coordinator`, because `test-utils` is where the `getOrderHashHex()` was implementation moved to.